README.md

```diff
@@ -78,20 +62,44 @@ SSL_CERT: /tmp/ssl-cert.pem
 ```
 
 #### Push with `meta-buildpack`
+**For custom buildpacks see [below](#custom-buildpack-instructions)**
 
-As described in the [`meta-buildpack` documentation](https://github.com/cf-platform-eng/meta-buildpack#how-it-works) the Conjur decorate buildpack will **ONLY** be invoked in the following scenarios:
+As described in the [`meta-buildpack` documentation](https://github.com/cf-platform-eng/meta-buildpack#how-it-works) the Conjur decorate buildpack will **ONLY** be invoked in the following are true:
```

Comment:
I actually don't see anything about manifests or `cf push -b` in the meta-buildpack docs; the only thing that it says is the meta-buildpack has to be at the top of the buildpacks list if you want it to call any decorator buildpacks.
If we have a reference for this, I think we should include it - otherwise I am in favor with adding the info about `meta-buildpack` being at the top to another section, moving the info on line 73 to another section, and deleting the rest of this content - because I think without a real reference, it's just sort of confusing.